src/test/java/com/jcabi/aether/MavenClasspathTest.java

```diff
@@ -55,35 +56,76 @@
 public final class MavenClasspathTest {
 
     /**
+     * Group of test artifact.
+     */
+    private static final String GROUP = "junit";
```

Comment:
@amihaiemil
>its just a constant to avoid checktyle complaining about duplicate strings

Sure, but please read http://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html